README.md

```diff
@@ -1,33 +1,62 @@
-# Project Name
-Project description
+# Usage Data
 
-# Disclaimer
-No Support and No Warranty are provided by SMA Technologies for this project and related material. The use of this project's files is on your own risk.
+This application is a tool designed to make it simple to provide SMA with daily task counts that a customer runs in their OpCon environments. It runs as an OpCon job (daily, by default) that can be configured like any other job.
 
-SMA Technologies assumes no liability for damage caused by the usage of any of the files offered here via this Github repository.
+By default, it uses both, an Azure cloud database (for SMA) and a local JSON file (for customer) to publish the task counts. However, it is configurable to turn off publishing at either of those locations.
```

Comment:
I'd simplify this line.  e.g.

By default, it publishes the data to an Azure service and a local JSON file.  It is configurable to turn off either publish location.